dev/NavigationView/NavigationViewItemsFactory.cpp

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "NavigationViewItemsFactory.h"
+#include "NavigationViewItemBase.h"
+#include "NavigationViewItem.h"
+
+NavigationViewItemsFactory::NavigationViewItemsFactory()
+{
+}
+
+winrt::UIElement NavigationViewItemsFactory::GetElementCore(winrt::ElementFactoryGetArgs const& args)
+{
+    winrt::IInspectable newContent = args.Data();
+    // Attempt to get NavigationViewItemBase from user defined IElementFactory
+    if (m_userElementFactory)
+    {
+        // Convert to args argument in the Windows::UI::Xaml namespace
+        m_getArgsWUX.Data(args.Data());
```

Comment:
>m_getArgsWUX [](start = 8, length = 12)

I don't know why you need to convert this. Please not that this Windows.UI.Xaml.ElementFactoryGetArgs are introduced in RS5, and I guess it will cause problem in old OS